operator-common/src/main/java/io/strimzi/operator/common/operator/resource/AbstractScalableResourceOperator.java

```diff
@@ -32,6 +32,7 @@
 
     public static final String STRIMZI_CLUSTER_OPERATOR_DOMAIN = "operator.strimzi.io";
     public static final String ANNOTATION_GENERATION = STRIMZI_CLUSTER_OPERATOR_DOMAIN + "/generation";
+    public static final String ANNOTATION_MANUAL_RESTART = "cluster." + STRIMZI_CLUSTER_OPERATOR_DOMAIN + "/manual-restart";
```

Comment:
I agree with Paolo - why both with the name (`strimzi.io/manual-rolling-update` sounds better) as well as with the `cluster.` prefix.